03week/ticTacToe.js

```diff
@@ -1,93 +1,72 @@
-'use strict';
-
-const assert = require('assert');
-const readline = require('readline');
-const rl = readline.createInterface({
-  input: process.stdin,
-  output: process.stdout
-});
-let board = [
-  [' ', ' ', ' '],
-  [' ', ' ', ' '],
-  [' ', ' ', ' ']
-];
-
-let playerTurn = 'X';
-
-function printBoard() {
-  console.log('   0  1  2');
-  console.log('0 ' + board[0].join(' | '));
-  console.log('  ---------');
-  console.log('1 ' + board[1].join(' | '));
-  console.log('  ---------');
-  console.log('2 ' + board[2].join(' | '));
-}
-
 function horizontalWin() {
   // Your code here
+  if(board[0][0] && board[0][1] && board[0][2]=='X'){
```

Comment:
You're repeating yourself A LOT here. You a) could just check for playerTurn instead of both letters b) just return the evaluation. The parent function, depending on the true/false value returned from this function should return something to the user